app/frontend/src/components/NotificationBar/_NotificationBar_Close.scss

```diff
@@ -0,0 +1,97 @@
+// External Imports
+@use "sass:map";
+
+// Internal Imports
+@use "../Basics" as *;
+
+/*****************************
+*** Notification Bar class ***
+*****************************/
+$themes: "primary", "success", "warning", "danger";
+
+p {
+  padding: 0;
+  margin: 0;
+}
+
+.notification-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1em;
+}
+
+.notification-bar-close {
+  cursor: pointer;
+  border: none;
+  background: none;
+  margin-right: auto;
+}
+
+.notification-bar-fade-appear {
+  opacity: 0;
+}
+
+.notification-bar-fade-appear-active {
+  opacity: 1;
+  transition-duration: 300ms;
+}
+
+.notification-bar-fade-enter {
+  opacity: 0;
+}
+
+.notification-bar-fade-enter-active {
+  opacity: 1;
+  transition-duration: 300ms;
+}
+
+.notification-bar-fade-exit {
+  opacity: 1;
+}
+
+.notification-bar-fade-exit-active {
+  opacity: 0;
+  transition-duration: 300ms;
+}
+
+.text {
```

Comment:
This name is too generic, so it increases the possibility of being overridden if we're careless in the future.